### DISCUSSION TITLE
Daily 14/04

### DISCUSSION THREAD

**Discussion by NataliaAlvarezIspc** [NataliaAlvarezIspc/proyecto-ispc-ecommerce#56] [Daily Sprint]
- Que hice ayer?
- Que haré hoy?
- Problemas o Impedimentos.

**Comment by NataliaAlvarezIspc** [comment]
Ayer terminé de instalar Typescript e instalé Angular. Complete la primer versión del IEEE830. Por la tarde participé en la clase sincrónica y por la noche de la reunión del equipo de trabajo. También mandé el formulario de presentación del grupo. 
Hoy seguiré con la sección de crear proyectos en Angular. Trataré de avanzar, también, en los contenidos de programación y desarrollo de software. También estaré revisando las historias de usuarios del repositorio.
Dificultades: falta de conocimiento que hace que todo me lleve mas tiempo.

**Comment by rpgrca** [comment]
Ayer nos juntamos para terminar de definir lo de la base de datos así que hoy empiezo con el diagrama de clase. Casi terminé con el apunte de angular, y viendo qué más hay en el de POO.

**Comment by BraianTroncoso** [comment]
Ayer tuve un examen aparte de Ingles, estuve en el meet ayudando a definir la BDD, y terminé el apunte de angular.
También renegé para instalar en ubuntu Angular por casi 2hs por el tema de las versiones de node, npm, y fué muy embolante.
Hoy voy a practicar realizar algun proyecto con Angular y seguir con OOP.

**Comment by AhumadaCeleste** [comment]
Ayer inicie con el materia  e instalaciones. aun no o logro terminarlo. Participe de la Daily con el grupo en donde definimos BDD,  Hoy tuve un día de poca disponibilidad por cuestiones familiares, pero asistiré a la clase y espero completar  lo pendiente de instalar para comenzar a practicar con todos los recursos. Mañana espero en algún  momento poder cerrar las issue que no realizaremos por el momento y avanzar con alguna tarea.

**Comment by KAAcevedo** [comment]
Estuve pensando y evaluando distintas User Stories un tanto diferentes a las dispuestas en la lista de Issues y las volqué en la sección de Discussions

**Comment by Brian-Almada** [comment]
Ayer seguí con el estudio de Angular. Hoy continúo en eso pero ya me topé con algunos problemas de paquetes que me están trabando el avance. Espero poder darle una solución.

**Comment by Tarias89** [comment]
Ayer no pude participar de la reunión por cuestiones personales, pude terminar de instalar angular.
Hoy tuve que dedicarme a tareas de otra tecnicatura que tienen finalización en dos días.
Impedimento: falta de tiempo para practicar y falta de conocimiento.
